commands.py

```diff
@@ -238,6 +238,40 @@ def deactivate_environment(self, index):
                 sublime.status_message('No active conda environment')
 
 
+class OpenCondaReplCommand(CondaCommand):
+    """Open a REPL tab within the activated Conda environment."""
+
+    def run(self, open_file='$file'):
+        """Display 'Conda: Open REPL' in Sublime Text's command palette.
+
+        When 'Conda: Open REPL' is clicked by the user, a new tab is opened with a REPL of the opened file in the current environment.
+        """
+        environment_path = self.project_data['conda_environment']
+        executable_path = "{}\\{}".format(os.path.expanduser(environment_path), "python")
```

Comment:
Unfortunately this \\ on line 250 will cause problems on Linux systems. You'll want to use `os.sep` here for a cross-platform separator.

Reply:
I think this could be `os.path.join(os.path.expanduser(environment_path, "python")`.